db/seeds.rb

```diff
@@ -245,43 +239,43 @@ def describe_teacher(teacher)
 
 grove_artisan_partnership_2021 = SchoolPartnership.create!(
```

Comment:
I think we might need to rename the partnerships here and everywhere else to the new names?